site/_docs/hive-config.md

```diff
@@ -12,13 +12,13 @@ with the same options.
 
 Key                      | Default     | Notes
 :----------------------- | :---------- | :------------------------
-orc.compress             | ZLIB        | high level compression = {NONE, ZLIB, SNAPPY}
+orc.compress             | ZLIB        | high level compression = {NONE, ZLIB, SNAPPY, LZO, LZ4, ZSTD}
```

Comment:
Could you add the JIRA IDs of these too, @cxzl25 ?

Reply:
Like this?
```
orc.compress             | ZLIB        | high level compression = {NONE, ZLIB, SNAPPY, LZO, LZ4 (ORC-77), ZSTD (ORC-363)}
```

Reply:
No~ What I meant was the PR description. Let me add that to the PR description and merge this.

Reply:
Thanks your help. @dongjoon-hyun